components/api/services/containers/rootfs/v1/rootfs.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package moby.core.services.containers.rootfs.v1;
+
+option go_package = "rootfs";
+
+import "github.com/moby/moby/components/api/types/container/container.proto";
+import "github.com/moby/moby/components/api/types/file.proto";
+
+service RootfsService {
```

Comment:
Can we just drop this service and expose bare containerd snapshotter service?
https://github.com/containerd/containerd/blob/master/api/services/snapshot/v1/snapshots.proto

That would deduplicate much effort for image&snapshot&rootfs abstraction.

cc @stevvooe @dmcgowan

Reply:
Mostly this is for compatibility for abstracting docker v1 compatibility into a nice service that's not dependent on the backend implementation.
We could expose the bare containerd service and just say "moby is built on containerd end of story".